src/services/registration.js

```diff
@@ -13,4 +13,14 @@ const create = async ({ questions, preferred_topics }) => {
   return response.data.registration
 }
 
-export default { create }
+const getOwn = async () => {
+  const response = await axios.get(url, {
+    headers: { Authorization: 'Bearer ' + getUserToken() }
+  })
+  if (response.status === 204) {
```

Comment:
When does the backend return 204?

Reply:
Backen returns 204 when there is no registration to current iteration by user fetching his/her own registration details. Yes, it is a bit unorthodox to responds in this way, but it suppressess 404 error from browser console this way.

Do you have a suggestion to this?

Reply:
Alright, I'd just add a comment about that there.